### ISSUE TITLE
Worker local storage

### ISSUE THREAD

**Issue by VoxSciurorum** [OpenCilk/cheetah#2]
We have a user request for a function to get the worker number, usable as an array index, but it's really a request for worker local storage.

**Comment by neboat** [comment]
Indeed, and I've heard similar requests through other channels from current Cilk Plus users whose codes use worker-local storage.  I think we'll need to address this issue sooner rather than later.
